stable/spinnaker/templates/configmap/service-settings.yaml

```diff
@@ -31,3 +34,11 @@ data:
       useExecHealthCheck: false
       serviceType: NodePort
     {{- end }}
+    {{- if .Values.halyard.additionalServiceSettings.deck.kubernetes }}
+{{ toYaml .Values.halyard.additionalServiceSettings.deck.kubernetes | indent 6 }}
+    {{- end }}
+  {{- if .Values.halyard.additionalServiceSettings.front50.kubernetes }}
+  front50.yml: |-
+    kubernetes:
+{{ toYaml .Values.halyard.additionalServiceSettings.front50.kubernetes | indent 6 }}
+  {{- end }}
```

Comment:
Can you add a new line here please

Reply:
https://github.com/helm/charts/pull/11631/commits/542ef7f0ca14a215a93b9b6566d66511a0031783